recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<36]
```

Comment:
why removing this skip here?

Reply:
It still uses py27 https://github.com/python-rope/rope/blob/b164d8e7baf5a713b524c784ab9e54462236e817/setup.py#L21